### DISCUSSION TITLE
Collabora Breaks File Preview/Icon

### DISCUSSION THREAD

**Discussion by imfbsbn** [nextcloud/all-in-one#3378] [Questions] [answered]
Running latest AIO v7.2.1 with all containers up to date. (Although have experienced this issue with v7.0.0 & v7.1.1 as well.)

Ubuntu 22.04 LTS

System works very well when the only selected container is NextCloud Talk. I've never tested with ClamAV, fulltext search, or the recording server. I have tested both with and without Imaginary; but let's put that off to the side for now. The bottom line is that everything is working: able to connect mobile devices, have online video calls with 7-8 people, attach external S3 storage, etc.

However, once I add and start the Collabora container *AND* edit a file NC can no longer seem to fetch a preview/icon for that file. As a result the "dashboard" and "files" pages will not complete loading. The page loads but the dial keeps spinning. When I press escape – to stop the page loading – I get a ? in place of the file preview/icon. See attached image.

Once the Collabora container is stopped, unchecked, and the system containers restarted everything returns to normal.

In logging, I get this error:
GuzzleHttp\Exception\ConnectException: cURL error 35: OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to nc.mydomain.com:443 (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://nc.mydomain.com/hosting/capabilities

Output of that file is:
{"convert-to":{"available":false},"hasMobileSupport":true,"hasProxyPrefix":false,"hasTemplateSaveAs":false,"hasTemplateSource":true,"hasZoteroSupport":true,"productName":"Collabora Online Development
Edition","productVersion":"23.05.3.1","productVersionHash":"a719c0f","serverId":"-------"}

Collabora docker log file attached.

Many thanks for any guidance in advance.

![Screenshot 2023-09-19 165044](https://github.com/nextcloud/all-in-one/assets/143741939/decace8a-6118-42c5-8b5f-8a09e57229d7)
[api_docker_logs_id=nextcloud-aio-collabora.txt](https://github.com/nextcloud/all-in-one/files/12666919/api_docker_logs_id.nextcloud-aio-collabora.txt)

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Comment by mangabs** [comment]
Hi,

I have exactly the same problem.
Running latest AIO 7.2.1 with all containers up to date.
Same problem with beta channel AIO 7.3.0 and all containers up to date.
I follow #1358 without success.

Here is the logs of the curl command (curl -vvv https://$NC_DOMAIN:443/hosting/discovery) inside the nextcloud container and collabora container logs.

Many thanks for your help.

[20230922-collabora.log](https://github.com/nextcloud/all-in-one/files/12704466/20230922-collabora.log)
[20230922-curl.log](https://github.com/nextcloud/all-in-one/files/12704467/20230922-curl.log)

**Reply by szaimen** [reply, marked as answer]
Okay, looks like this needs to be fixed in nextcloud/server then

**Reply by devdracon** [reply]
Anything new here? The Problem still exists with AIO 12.0.0. Beta...
